tests/test_0652_dask-for-awkward.py

```diff
@@ -0,0 +1,96 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/uproot4/blob/main/LICENSE
+
+import numpy
+import pytest
+import skhep_testdata
+
+import uproot
+
+dask = pytest.importorskip("dask")
+dask_awkward = pytest.importorskip("dask_awkward")
+
+
+def test_single_dask_awkward_array():
+    test_path = skhep_testdata.data_path("uproot-Zmumu.root") + ":events"
+    ttree = uproot.open(test_path)
+
+    ak_array = ttree.arrays()
+    dak_array = uproot.dask(test_path, library="ak")
+
+    assert len(ak_array) == len(dak_array)
+
+    assert dak_array.compute().to_list() == ak_array.to_list()
```

Comment:
When comparing a dask-awkward array to a concrete awkward array try to use [dask-awkward's `assert_eq`](https://github.com/ContinuumIO/dask-awkward/blob/main/src/dask_awkward/testutils.py#L18); it does more checks than just for equality